FE/todo-app/src/app/page.tsx

```diff
@@ -1,25 +1,20 @@
 "use client";
-import { useGetToDosQuery } from "@/store/api/todoApi";
 import { ToDo } from "../components/getToDo/ToDo";
 import styles from "./page.module.css";
 import { useModal } from "@/hooks/useModal";
-import { useAppSelector } from "@/store/hooks";
+import { useFilteredToDos } from "../hooks/useFilterToDo";
 
 export default function Home() {
-  const { data } = useGetToDosQuery();
   const { isOpen, open, close } = useModal();
-
-  const { idGroup } = useAppSelector(({ idGroupToDo }) => idGroupToDo);
+  const { useFilteredToDo } = useFilteredToDos();
```

Comment:
ten nazev exportuj nějak rozumněji, tahkle to navadí že exportuješ nějaký hook místo dat.. pojmenuj to třeba filteredTodos